components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx

```diff
@@ -176,4 +156,20 @@ class EIAnalyticsSearchBox extends Widget {
     }
 }
 
+function getCurrentPage() {
```

Comment:
Why these function resides outside of a class?

Reply:
Fixed at b233d5335fc34cc9a03e24a4944cd39d0f6c3677